src/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@styled-system/prop-types';
```

Comment:
This is just the standard manual module declaration for a package with missing types.